lib/chrome_wrapper.ts

```diff
@@ -522,6 +522,8 @@ export default class ChromeWrapper {
 
     const browser = await this.browser
     const page = await browser.newPage()
+    await page.setDefaultNavigationTimeout(0)
```

Comment:
Hmm I wonder if we should just make the timeout really long like 10min. Could removing the timeout lead to tests running forever in ci?